Automattic/wp-calypso
# Fix hidden site count in site selector for Jetpack cloud

### Changes proposed in this Pull Request

In Jetpack cloud, the count of hidden sites in the site selector reflects the total number of hidden sites. It should take into account Jetpack sites only.

Fixes 1164141197617539-as-1199632056318075

### Testing instructions

To reproduce the issue:
- Make sure you have both standard and self-hosted Jetpack hidden sites
- Visit `https://cloud.jetpack.com/landing`
- Notice that the number of hidden sites doesn't match the number of Jetpack hidden sites

To test the PR:
- Download the PR and run both Calypso and cloud concurrently
- Visit cloud and verify that the number matches the actual number of Jetpack hidden sites
- Visit `calypso.localhost:3000/home` and verify that the number matches the total number of hidden sites

### Screenshots

_Site selector_
<img width="565" alt="Screen Shot 2021-02-10 at 4 49 06 PM" src="https://user-images.githubusercontent.com/1620183/107577382-5cd09b00-6bc0-11eb-8b6a-cb2362541738.png">
